database/sql/enterprise_test.go

```diff
@@ -238,16 +238,16 @@ func (s *EnterpriseTestSuite) TestGetEnterpriseNotFound() {
 }
 
 func (s *EnterpriseTestSuite) TestGetEnterpriseDBDecryptingErr() {
-	s.Fixtures.SQLMock.
-		ExpectQuery(regexp.QuoteMeta("SELECT * FROM `enterprises` WHERE name = ? COLLATE NOCASE AND `enterprises`.`deleted_at` IS NULL ORDER BY `enterprises`.`id` LIMIT 1")).
-		WithArgs(s.Fixtures.Enterprises[0].Name).
-		WillReturnRows(sqlmock.NewRows([]string{"name"}).AddRow(s.Fixtures.Enterprises[0].Name))
+	// s.Fixtures.SQLMock.
+	//	ExpectQuery(regexp.QuoteMeta("SELECT * FROM `enterprises` WHERE name = ? COLLATE NOCASE AND `enterprises`.`deleted_at` IS NULL ORDER BY `enterprises`.`id` LIMIT 1")).
+	//	WithArgs(s.Fixtures.Enterprises[0].Name).
+	//	WillReturnRows(sqlmock.NewRows([]string{"name"}).AddRow(s.Fixtures.Enterprises[0].Name))
 
-	_, err := s.StoreSQLMocked.GetEnterprise(context.Background(), s.Fixtures.Enterprises[0].Name)
+	//_, err := s.StoreSQLMocked.GetEnterprise(context.Background(), s.Fixtures.Enterprises[0].Name)
 
-	s.assertSQLMockExpectations()
-	s.Require().NotNil(err)
-	s.Require().Equal("decrypting secret: failed to decrypt text", err.Error())
+	// s.assertSQLMockExpectations()
+	// s.Require().NotNil(err)
+	// s.Require().Equal("decrypting secret: failed to decrypt text", err.Error())
```

Comment:
The test expects an error for an empty webhook secret in the database. The [validateHookBody](https://github.com/cloudbase/garm/blob/main/runner/runner.go#L544-L547) func seems to imply that empty secrets disables validation. IMHO the implementation is contradictory - this is why I would need your decision on the topic before being able to continue with the implementation

Reply:
You are correct. I think we should make signature validation a hard requirement.

Reply:
Yes, that would have been my decision, too. I think I'll manage to adopt the code beginning of next week. I'll let you know as soon as I am done.

Reply:
That sounds great! Thanks!